tendrl/gluster_integration/sds_sync/cluster_status.py

```diff
@@ -54,6 +54,28 @@ def sync_cluster_status(volumes):
         if not connected:
             is_healthy = False
 
+    cluster_gd = NS.gluster.objects.GlobalDetails().load()
+    old_status = cluster_gd.status
+    curr_status = 'healthy' if is_healthy else 'unhealthy'
+    if curr_status != old_status:
+        msg = ("Status of cluster: %s "
```

Comment:
"Health status of Cluster" instead of "Status of cluster"

Reply:
Done